test/useMediaQuery_test.tsx

```diff
@@ -117,7 +117,7 @@ describe('useMediaQuery', () => {
 
   it('uses query prop if it has one', () => {
     (window as unknown as MockWindow).matchMedia.setConfig({
-      orientation: 'landscape'
```

Comment:
orientation isn't supported by the types for mock matchmedia although the functionality did pass when cast to any